### ISSUE TITLE
New language dutch

### ISSUE THREAD

**Issue by Webserve** [sbidy/privacyIDEA-ADFSProvider#40]
Today i configured the adfs but was hitting the following error:

https://www.securemfa.com/knowledge-base/no-style-sheet-is-configured-in-the-active-theme-for-default-locale

So i created the dutch translation. Mayby you can add this to the source:

config.xml
```
  <interface LICD="1043"> 
     <!-- nl-NL -->
     <errormessage>Inloggen mislukt! Probeer het opnieuw!</errormessage>
     <welcomemessage>Voer uw eenmalige code in:</welcomemessage>
     <titel>privacyIDEA OTP Adapter - NL</titel>
  </interface>

```

AdapterMetadata.cs
add:
```
new CultureInfo("nl-NL").LCID 

_friendlyNames.Add(new CultureInfo("nl-NL").LCID, "privacyIDEA ADFS authentication provider");

_descriptions.Add(new CultureInfo("nl-NL").LCID, "privacyIDEA ADFS provider om de API te bedienen");
```

**Comment by sbidy** [comment]
Thank you! The translations of several languages will come in the 1.3.7 - I'll add your dutch too!

Is the issue, mentioned in the link, still present in your system?

**Comment by Webserve** [comment]
After adding the dutch language this issue was resolved.

**Comment by sbidy** [comment]
Released with v 1.3.7
